### ISSUE TITLE
PLS-DA scaling method

### ISSUE THREAD

**Issue by dshirya** [bobleesj/SAF-CAF-performance#1]
## Description

After modifying the preprocessing pipeline to include normalization, a discrepancy was observed in the LV % (Latent Variable Variance) values compared to the previous implementation. The scatter plots of the PLS-DA remain visually identical, but the explained variance percentages have changed.

This issue arises due to the interplay between external scaling in preprocessing and the internal scaling mechanism of the PLS-DA model. Disabling the internal scaling in PLS-DA causes the raw feature variances to dominate, leading to different variance contributions by each latent variable and better results in cluster separation.

## Steps to Reproduce

1.  Modify the preprocessing (preprocess.py):
```
def standardize_data(X):
    """
    Normalizes the data by combining autoscaling and rescaling to [0, 1].
    """
    scaler_standard = StandardScaler()
    X_scaled = scaler_standard.fit_transform(X)

    scaler_minmax = MinMaxScaler()
    X_normalized = scaler_minmax.fit_transform(X_scaled)

    return X_normalized
```
2. Disable internal scaling in the PLSRegression model (PLS_DA_plot.py) by setting scale=False:
```
pls = PLSRegression(n_components=2, scale=False)
```
3. Compare the results (scatter plots and LV %) with:
	•	Internal scaling enabled (scale=True).
	•	The original implementation without normalization in preprocessing.

**Comment by bobleesj** [comment]
Thanks Danila!

**Comment by dshirya** [comment]
Update

There is no need to disable internal scaling. It goes by default even if you add `scale = False`. 
Because internal scaling is centering of the data and it is done in both cases `scale = False` and `scale = True`

**Comment by bobleesj** [comment]
Thanks @dshirya for this update. Glad that it remains useful.
